lib/workers/qemu.js

```diff
@@ -59,12 +70,21 @@ module.exports = class QEMUWorker {
       '-smp', '4'
     ])
 
-    childKillEventHandler(this.child, 'exit')
-    childKillEventHandler(this.child, 'uncaughtException')
-    childKillEventHandler(this.child, 'unhandledRejection')
+    this.child.stderr.on('data', (data) => {
+      stderr += data
+    })
+
+    this.child.on('exit', (code) => {
+      if (code !== 0) {
+        throw new Error(stderr)
```

Comment:
Maybe we can say something like `QEMU exit with an error: ${stderr}`, as right now we would see a confusing empty error if QEMU didn't print anything to `stderr`, right?